js/app.js

```diff
@@ -0,0 +1,487 @@
+    playGame.textContent = "You Busted"
+    document.querySelector(".moveList").style.visibility="hidden"
+    // score = score - 1000 
+    outcome = "L"
+
+    setTimeout(clearPhase, 3000)
+  }
+}
+
+//function to make the 2nd drawn card for the dealer in the draw phase to show up as the back of the card instead of its front
+function dealerDrawPhase(){
+  drawCard()
+  dealerHand.push(cardPicked)
+  let newCard = document.createElement("div")
+  newCard.setAttribute("class", `card back-blue`)
+  dealerHandElem.appendChild(newCard)
+}
+
+function dealerHit(){
+  drawCard()
+  dealerHand.push(cardPicked)
+  dRender()
+  cardValue(dealerHand)
+  dTotalRender()
+}
+
+//function to add card value
+function cardValue(handArray){
+  totalHolder = 0
+  tempArray = [] // empty tempArray
+  tempArray = [...handArray] //fill tempArray
+  let isThereAnAce = false
+  tempArray.forEach(function(card){
+    valueHolder = card.slice(1)
+
+    if(isNaN(parseInt(valueHolder)) === false){
+      totalHolder = totalHolder + parseInt(valueHolder)
+      }
+    if(valueHolder === 'Q' || valueHolder === 'K' || valueHolder === 'J'){
+      totalHolder += 10
+    }
+    if(valueHolder === 'A'){
+      totalHolder+= 1
+      isThereAnAce = true
+    }
+    })
+  if(totalHolder < 12 && isThereAnAce === true){
+    totalHolder +=10
+  }
+  if(totalHolder > 21){
+    totalHolder = 'Bust'
+  }
+  
+}
+
+function double(){
+  bet = bet * 2
+  betArea.textContent=`Bet: ${bet}p`
+  renderChipDouble()
+  handleHit()
+  document.querySelector(".moveList").style.visibility="hidden"
+  document.querySelector("#double").style.visibility="hidden"
+  setTimeout(dealerPhase, 3000)
+}
+
+//1. If a player's first two cards are of the same denomination, such as two jacks or two sixes, they may choose to treat them as two separate hands when their turn comes around. 
+//2. The amount of the original bet then goes on one of the cards, and an equal amount must be placed as a bet on the other card.
+//3. The player first plays the hand to their left by standing or hitting one or more times; only then is the hand to the right played. The two hands are thus treated separately, and the dealer settles with each on its own merits. 
+//4. With a pair of aces, the player is given one card for each ace and may not draw again. Also, if a ten-card is dealt to one of these aces, the payoff is equal to the bet (not one and one-half to one, as with a blackjack at any other time).
+function splitPairs(){
+  let holder = playerHand.pop
+  playerSplitHand.push(holder)
+}
+
+function handleBet(num){
+  bet = num
+  if((score - bet) < 0){
+    playGame.textContent="Not Enough Chips"
+    document.querySelector(".betList").style.visibility="hidden"
+    setTimeout(init, 2000)
+  }else{
+    betArea.textContent=`Bet: ${bet}p`
+    renderChip()
+    document.querySelector(".betList").style.visibility="hidden"
+    setTimeout(drawHands, 1000)
+    }
+  }
+function renderChip(){
```

Comment:
You're really good on maintaining good indentation and vertical white space up to this point, but then things start to break down a little bit. Take 5 minutes to clean up your spacing, you won't regret it.